Sources/TuistGenerator/Mappers/ModuleMapMapper.swift

```diff
@@ -2,6 +2,35 @@ import Foundation
 import TSCBasic
 import TuistCore
 import TuistGraph
+import TuistSupport
+
+enum ModuleMapMapperError: FatalError {
+    case invalidTargetDependency(sourceProject: AbsolutePath, sourceTarget: String, dependentTarget: String)
+    case invalidProjectTargetDependency(sourceProject: AbsolutePath, sourceTarget: String, dependentProject: AbsolutePath, dependentTarget: String)
+
+    /// Error type.
+    var type: ErrorType {
+        switch self {
+        case .invalidTargetDependency, .invalidProjectTargetDependency: return .abort
+        }
+    }
+
+    /// Error description.
+    var description: String {
+        switch self {
+        case let .invalidTargetDependency(sourceProject, sourceTarget, dependentTarget):
+            return """
+            Invalid target dependency for target \(sourceTarget) of project at path \(sourceProject.pathString): \
```

Comment:
Nitpick:
```
Target A of the project at path X depends on a target B that can't be found
```
Do we know why that might be the case? If so, we can add an extra sentence hinting to the users what might be the issue.